dtreeviz/trees.py

```diff
@@ -859,6 +867,9 @@ def get_leaves():
     edges = []
     # non leaf edges with > and <=
     for node in get_internal_nodes():
+        if depth_range_to_display is not None:
+            if node.level not in range(depth_range_to_display[0], depth_range_to_display[1]):
```

Comment:
do we need `+1` ?

Reply:
this loop is for displaying the edges, so we dont need to display the edges for the last level

Reply:
ah!